packages/text/src/TextStyle.ts

```diff
@@ -57,6 +89,38 @@ const genericFontFamilies = [
  */
 export class TextStyle
```

Comment:
I think it should implement ITextStyle

Reply:
I agree. I originally had that there but for some reason I removed it and forgot to put it back in.